modules/gapi/include/opencv2/gapi/streaming/onevpl/utils.hpp

```diff
@@ -0,0 +1,37 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2022 Intel Corporation
+
+#ifndef OPENCV_GAPI_STREAMING_ONEVPL_UTILS_HPP
+#define OPENCV_GAPI_STREAMING_ONEVPL_UTILS_HPP
+
+#include <opencv2/gapi/streaming/source.hpp>
+#include <opencv2/gapi/streaming/onevpl/cfg_params.hpp>
+#include <opencv2/gapi/streaming/onevpl/data_provider_interface.hpp>
+#include <opencv2/gapi/streaming/onevpl/device_selector_interface.hpp>
+
+namespace cv {
+namespace gapi {
+namespace wip {
+namespace onevpl {
+
+/**
+ * @brief creates a G-API streaming source based on OneVPL with default configuration.
+ *
+ * The default configuration assumes utilizing GPU device, using ID3D11Device,
+ * ID3D11DeviceContext, mfxImplDescription.AccelerationMode and MFX_ACCEL_MODE_VIA_D3D11.
+ *
+ * @note stream sources are passed to G-API via shared pointers, so
+ *  please gapi::make_onevpl_src<> to create objects and ptr() to pass a
+ *  GSource to cv::gin().
+ */
+cv::Ptr<cv::gapi::wip::IStreamSource> make_default_onevpl_src(const std::string& input);
```

Comment:
hah given its content, I believe it really should be "default.hpp" :)

Reply:
Done